src/System.Diagnostics.TraceSource/tests/TraceSourceClassTests.cs

```diff
@@ -116,6 +116,44 @@ public void SwitchLevelTest(SourceLevels sourceLevel, TraceEventType messageLeve
             trace.TraceEvent(messageLevel, 0);
             Assert.Equal(expected, listener.GetCallCount(Method.TraceEvent));
         }
+
+        [Fact]
+        public void NullSourceName()
+        {
+            Assert.Throws<ArgumentNullException>(() => new TraceSource(null));
```

Comment:
....it's what the tests are there for, I have it in one of the other tests, and I even wrote a note after you mentioned it last time.
:facepalms
Argh.